articles/connections/social/google.md

```diff
@@ -14,6 +14,8 @@ seo_alias: google
 
 To connect your Auth0 client to Google and Google Apps, you will need to generate a *Client ID* and *Client Secret* in a Google project, copy these keys into your Auth0 settings, and enable the Connection.
 
+If you are looking to manage authentication in your native Android app [see the Android quickstart](/quickstart/native/android).
+
```

Comment:
@maria I am not sure all the quickstarts are relevant to all the connection docs, most of the social connections are setup after the quickstart steps are followed. With this specific change the user clicked on the Google social connection doc but was looking for google native android instructions and they were confused. I agree that it seems a but out of context but I'm not sure where else to put it?